pom.xml

```diff
@@ -75,6 +75,12 @@
         <apache.mime4j.version>0.6</apache.mime4j.version>
         <jboss.dmr.version>1.5.1.Final</jboss.dmr.version>
         <bouncycastle.version>1.68</bouncycastle.version>
+
+        <!-- TODO Are these correct versions? -->
+        <bouncycastle.fips.version>1.0.2</bouncycastle.fips.version>
+        <bouncycastle.pkixfips.version>1.0.3</bouncycastle.pkixfips.version>
+        <bouncycastle.tlsfips.version>1.0.11</bouncycastle.tlsfips.version>
```

Comment:
I used some newer versions here:

https://github.com/stianst/keycloak/blob/24f10231fd49db6c34fcca222d720b683f523069/quarkus/runtime/pom.xml#L526

Reply:
Updated per your versions. I guess BouncyCastle provides some mapping to compare BouncyCastle FIPS versions to BouncyCastle non-fips version. I wonder if we should investigate this further...